client/handle_http.go

```diff
@@ -2856,9 +2862,9 @@ Loop:
 	// <= 0, it indicates we don't know how large the transfer was supposed to be in the first
 	// place.
 	if totalSize > 0 {
-		if downloaded != totalSize {
-			log.WithFields(fields).Debugf("Download completed but received size %db does not match expected size '%db'", downloaded, totalSize)
-			err = errors.Errorf("download completed but received size %db does not match expected size '%db'", downloaded, totalSize)
+		if bytesSoFar+downloaded != totalSize {
+			log.WithFields(fields).Debugf("Download completed but received size %db does not match expected size '%db'", bytesSoFar+downloaded, totalSize)
+			err = errors.Errorf("download completed but received size %db does not match expected size '%db'", bytesSoFar+downloaded, totalSize)
```

Comment:
Can you fix a mistake I made that's now bothering me? One of the format specifiers is in single quotes and the other isn't. I don't have a strong preference either way, but it'd be better to be consistent.